controllers/src/main/java/com/redhat/ipaas/controllers/integration/StatusChangeHandlerProvider.java

```diff
@@ -37,26 +37,30 @@
 
         class StatusUpdate {
             private Optional<Integration.Status> newStatus;
-            private String message;
+            private Optional<String> message;
```

Comment:
Actually I don't meant the status object, think `newStatus` is  a good fit for a `StatusUpdate`. I meant to rename `message` --> `errorMessage` as it is supposed only be set in the error case.